smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/spring/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -21,8 +26,21 @@
     })
 class OtelSpringStarterSmokeTest extends AbstractOtelSpringStarterSmokeTest {
 
+  @Autowired RestClient.Builder restClientBuilder;
+  @LocalServerPort private int port;
+
   @Test
   void restClient() {
-    assertClient(OtelSpringStarterSmokeTestController.REST_CLIENT);
+    testing.clearAllExportedData();
+
+    RestClient client = restClientBuilder.baseUrl("http://localhost:" + port).build();
+    assertThat(
+            client
+                .get()
+                .uri(OtelSpringStarterSmokeTestController.PING)
+                .retrieve()
+                .body(String.class))
+        .isEqualTo("pong");
+    assertClient();
```

Comment:
I'm not sure the purpose of this test now since the `RestClient` telemetry is no longer captured or verified

Reply:
assertClient is doing the verification

Reply:
would it help to inline the method?

Reply:
oh, sorry! I got confused b/c I was thinking of the other smoke test pattern where the test code itself doesn't get instrumented, only the app code, but realize that's not the case for these smoke tests